routes/common/postgres_helper.rb

```diff
@@ -249,4 +255,20 @@ def failover
 
     Serializers::Postgres.serialize(@resource, {detailed: true})
   end
+
+  def send_notification_mail_to_partners(resource, user_email)
+    if resource.flavor == PostgresResource::Flavor::PARADEDB
+      Util.send_email(Config.postgres_paradedb_notification_email, "New ParadeDB Postgres database has been created.",
+        greeting: "Hello ParadeDB team,",
+        body: ["New ParadeDB Postgres database has been created.",
+          "Id: #{resource.id}",
```

Comment:
We don't show UUID anywhere. We can share UBID. It's more common in our team.

Reply:
I agree, let's stick with ubid. I suspect email notification will be replaced with an API eventually.

Reply:
It has thumbs up emoji and resolved but it's still `resource.id`

Reply:
Weird, I guess the change got lost while rebasing.